examples/intel/ip/ip.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+import pangea.exceptions as pe
+from pangea.config import PangeaConfig
+from pangea.services import IpIntel
+
+token = os.getenv("PANGEA_IP_INTEL_TOKEN")
+domain = os.getenv("PANGEA_DOMAIN")
+config = PangeaConfig(domain=domain)
+intel = IpIntel(token, config=config)
+
+
+def main():
+    print(f"Checking IP...")
+
+    try:
+        response = intel.lookup(ip="93.231.182.110", provider="crowdstrike", verbose=True, raw=True)
+        print(f"Response: {response.result}")
+    except pe.PangeaAPIException as e:
+        print(f"Request Error: {e.response.summary}")
+        for err in e.errors:
+            print(f"\t{err.detail} \n")
```

Comment:
nit: no need for trailing whitespace before the newline. also print adds a newline by default